Lib/test/test_os.py

```diff
@@ -352,6 +352,11 @@ def test_statvfs_attributes(self):
         for value, member in enumerate(members):
             self.assertEqual(getattr(result, 'f_' + member), result[value])
 
+        self.assertTrue(hasattr(result, 'f_fsid'))
```

Comment:
Should probably verify that the value is an integer as well:

~~~~~~ python
self.assertIsInstance(result.f_fsid, int)
~~~~~~